apps/dialtone-documentation/docs/components/scrollbar.md

```diff
@@ -0,0 +1,190 @@
+---
+title: Scrollbar
+description: A directive that adds a custom overlay scrollbar to any scrollable region.
+status: beta
+thumb: true
+image: assets/images/components/scroller.png
+---
+
+## Scrollbar directive
+
+Allows to add overlay scrollbars that will look the same for every browser. The directive sets up the scrollbars from the library [OverlayScrollbars](https://kingsora.github.io/OverlayScrollbars/).
+
+<code-well-header>
+  <dt-stack class="d-hmx164 d-w30p d-bar8 d-ba d-bc-default" v-dt-scrollbar>
+    <div v-for="item in items" class="item">
+      {{ item}}
+    </div>
+  </dt-stack>
```

Comment:
Hmm, shucks. We just created a great example of the **Limitations** you documented. In this case, with `<dt-stack>`

<img width="586" alt="image" src="https://github.com/dialpad/dialtone/assets/1165933/1aeb4bba-2c01-420b-b60c-954ec0e57892">

Perhaps DtStack should be wrapped? In fact, maybe the recommendation is to default to wrapping with `<div v-dt-scrollbar>` rather than attaching `v-dt-scrollbar` to an existing element?

```html
<div v-dt-scrollbar>
  <dt-stack ...>
    ...
  </dt-stack>
</div>
```

Reply:
I pulled it down locally to test my assumption of wrapping with `<div v-dt-scrollbar>`, but that didn't immediately play well. So yeah, sometimes that might work. Other times you'll have to figure the right markup structure. This one did it fine:

```html
<div class="d-bar8 d-ba d-bc-default d-hmx164 d-w30p" v-dt-scrollbar>
  <dt-stack>
    <div v-for="item in items" class="item">
      {{ item}}
    </div>
  </dt-stack>
</div>
```